apigw-http-api-step-functions-express-java/example-pattern.json

```diff
@@ -0,0 +1,48 @@
+{
+  "title": "API Gateway to Step Functions",
```

Comment:
add "API Gateway HTTP API to Step Functions"

Reply:
Done